tutorials/aml/notebooks/scripts/score.py

```diff
@@ -31,31 +31,27 @@ def score_pipeline(hotelActivityData, customerData, serviceUsageData, output_dat
 run = Run.get_context()
 ws = run.experiment.workspace
 
-print("Argument 1: %s" % args.input_data1)
-print("Argument 2: %s" % args.input_data2)
-print("Argument 3: %s" % args.input_data3)
-
-# get the input dataset by ID
-from azureml.core import Run
-
+print("geting datasets ...")
 hotel_activity_data = Run.get_context().input_datasets['HotelStayActivity_dataset']
 customer = Run.get_context().input_datasets['Customer_dataset']
 service_usage = Run.get_context().input_datasets['ServiceUsage_dataset']
 
 print("Output Location", args.output_datastore + args.output_path)
 
 # Load Model 
-print("loading model")
-print("get directory", os.getcwd())
-
+print("loading model ...")
 model_path = Model.get_model_path("churnscore", _workspace = ws,version = 1)
 print("model_path : ", model_path)
 model = joblib.load(model_path)
 
-#print("Argument 2: %s" % args.output)                       
 
-#if not (args.output is None):
-#    os.makedirs(args.output, exist_ok=True)
-#    print("%s created" % args.output)
+config = {
+    "output_datastore" : args.output_datastore,
+    "output_path" : args.output_path,
+    "model" : model,
+    "run" : run,
+    "workspace": ws,
```

Comment:
Do we need the workspace in the config ? We can just get it from the run run.Experiment.Workspace ?

Reply:
We have a common library function that is used for both train and score. Train is implemented in a jupyter notebook and does not have run parameter. This is why we pass workspace in the config instead of using run.experiment.config.